IOPool/Input/test/BuildFile.xml

```diff
@@ -25,4 +25,7 @@
   <test name="TestIOPoolInputNoParentDictionary" command="testNoParentDictionary.sh"/>
   <test name="TestFileOpenErrorExitCode" command="testFileOpenErrorExitCode.sh"/>
   <test name="TestIOPoolInputSchemaEvolution" command="testSchemaEvolution.sh"/>
+
+  <test name="TestIOPoolInputRefProductIDMetadataConsistency" command="run_TestRefProductIDMetadataConsistencyRoot.sh"/>
```

Comment:
Why a .sh instead of having the tests run cmsRun directly?

Reply:
Partially out of old habit, but running the four dependent `cmsRun` jobs in a script feels easier than expressing the DAG in the `BuildFile.xml`?